src/main/java/org/example/userproject1/endpoint/UserController.java

```diff
@@ -0,0 +1,86 @@
+package org.example.userproject1.endpoint;
+
+import lombok.RequiredArgsConstructor;
+import org.example.userproject1.entity.User;
+import org.example.userproject1.service.PhoneServise;
+import org.example.userproject1.service.UserSevise;
+import org.example.userproject1.validator.UserValidator;
+import org.example.userproject1.validator.ValidationResult;
+import org.example.userproject1.validator.Validator;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.view.RedirectView;
+
+@Controller
+@RequiredArgsConstructor
+@RequestMapping("/user")
+public class UserController {
+
+    private final UserSevise userSevise;
+    private final UserValidator userValidator;
+
+    @GetMapping("")
+    public ModelAndView list() {
+        ModelAndView result = new ModelAndView("allUserPage");
+        result.addObject("UserList", userSevise.listAll());
+        return result;
+    }
+
+
+    @GetMapping("/create")
+    public ModelAndView create() {
+        ModelAndView result = new ModelAndView("createUserPage");
```

Comment:
Just curious why do you need 2 endpoints here for create/edit.

Reply:
I thought if the project was going to grow, it would be better to make different endpoints.